llvm/lib/Target/AMDGPU/AMDGPUISelDAGToDAG.h

```diff
@@ -107,10 +107,6 @@ class AMDGPUDAGToDAGISel : public SelectionDAGISel {
   std::pair<SDValue, SDValue> foldFrameIndex(SDValue N) const;
   bool isInlineImmediate(const SDNode *N) const;
 
-  bool isInlineImmediate16(int64_t Imm) const {
```

Comment:
What's the plus of removing this if there's no change to the underlying helper function?

Reply:
The change of `AMDGPU::isInlinableLiteral16` will come soon and then there is no sufficient information here to call the new function. Besides, these functions are literally unused. I don't see any point of keeping them.